boards/xtensa/esp32/esp32-audio-kit/src/esp32-audio-kit.h

```diff
@@ -0,0 +1,177 @@
+/****************************************************************************
+ * boards/xtensa/esp32/esp32-audio-kit/src/esp32-audio-kit.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __BOARDS_XTENSA_ESP32_ESP32_AUDIO_KIT_SRC_ESP32_AUDIO_KIT_H
+#define __BOARDS_XTENSA_ESP32_ESP32_AUDIO_KIT_SRC_ESP32_AUDIO_KIT_H
+
+/****************************************************************************
+ * Included Files
+ ****************************************************************************/
+
+#include <nuttx/config.h>
+#include <nuttx/compiler.h>
+#include <stdint.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+/* ESP32-LyraT GPIOs ********************************************************/
+
+/* LEDs */
+
+#define GPIO_LED1  22
+
+/* Audio Amplifier */
+
+#define SPEAKER_ENABLE_GPIO   21
+
+/* Buttons */
+
+/* As BOOT_BUTTON shares pins with I2S and the SD card it cannot be used
+ * as an user button like regular ESP32 boards. As the ESP32-LyraT has
+ * other buttons that can be used, BOOT_BUTTON is disabled in the buttons
+ * driver to avoid conflict with the remaining peripherals.
+ */
+
+#define BUTTON_REC                  36
+#define BUTTON_MODE                 39
+#define BUTTON_PLAY_TP_CHANNEL      8
+#define BUTTON_SET_TP_CHANNEL       9
+#define BUTTON_VOLM_TP_CHANNEL      4
+#define BUTTON_VOLP_TP_CHANNEL      7
```

Comment:
This section was made for the ESP32-LyraT board. Please double check the pins and buttons.

Reply:
Fixed! I decided to remove buttons support for now